### PR TITLE
Open Rotten Tomatoes in new tab

### PR DESCRIPTION
**Pull request by rakleed**
Most of the time, if I want to see the details of Rotten Tomatoes ratings and reviews, I don't want IMDb closed.

### CODE REVIEW

**Comment by chocolateboy** [comment]
Thanks for the PR!

> I don't want IMDb closed

Isn't this what right click is for?

I don't like this feature (and consider it a [misfeature](https://stackoverflow.com/a/3907140)), but — on the other hand — it looks like IMDb does the same for external links (e.g. for [Metacritic](https://www.imdb.com/title/tt0308208/criticreviews)), so it would be reasonable to add it for compatibility since I'm not trying to blaze a new UX.

Would be nice to hear what others think since I disable it and therefore won't benefit from it or be [annoyed](https://stackoverflow.com/a/946263) by it.

**Comment by rakleed** [comment]
> Isn't this what right click is for?

On the one hand, you are right, but at the same time I need to do three clicks instead of one. And even if I use the mouse wheel, I still need to go to the open tab after that. If I click on a link with the `target="_blank"` attribute, then the new tab is immediately active for me.

> it looks like IMDb does the same for external links (e.g. for Metacritic), so it would be reasonable to add it for compatibility since I'm not trying to blaze a new UX.

Yes, this is also one of the reasons why I added a force open link to Rotten Tomatoes in a new tab.

File: src/imdb-tomatoes.user.js [hunk 1]
```diff
@@ -825,7 +825,7 @@ function addWidget ($ratings, $imdbRating, { consensus, rating, url }) {
     // 7) update the link's label and URL
     $rtRating
         .find('a[role="button"]')
-        .attr({ 'aria-label': 'View RT Rating', href: url })
+        .attr({ 'aria-label': 'View RT Rating', href: url, target: _blank })
```
Comment (chocolateboy) [inline src/imdb-tomatoes.user.js:828]: Typo: `{ target: _blank }` -> `{ target: '_blank' }`

Comment (rakleed) [reply src/imdb-tomatoes.user.js]: Fixed.

**Comment by chocolateboy** [comment]
Added an option for this in v5.1.0:

imdb-tomatoes-link-target-option

**Comment by rakleed** [comment]
@chocolateboy but by default, do you still think it's better to use opening in the same tab?

**Pull request closed without merging**